packages/apps/composer-app/src/playwright/first-run.spec.ts

```diff
@@ -0,0 +1,27 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import { test } from '@playwright/test';
+import { expect } from 'chai';
+
+import { AppManager } from './app-manager';
+
+test.describe('First-run tests', () => {
```

Comment:
for this I was thinking that it'd be good to also have a test which stepped through and made sure that each step rendered properly. In the past I think the joyride stopped working because it lost some of it's anchors as the UI shifted